scripts/generate_schema/datagov/README.md

```diff
@@ -1,6 +1,10 @@
 # datagov README
 
-1. output0.jsonl is one of file I crawled from website data.gov
-2. crawl.py is the python script to crawl metadata from data.gov
-3. generate_datagov_schema.py is the python script to convert the old schema to the new schema we need
+1. crawl.py is the python script to crawl metadata from data.gov
+
+   ​	**python3 crawl.py \<the folder you want to store the files>**
+
+2. generate_datagov_schema.py is the python script to convert the old schema to the new schema we need
+
+   ​	**python2 generate_datagov_schema.py \<the folder you store the original files> \<the folder you store the generate files >**
```

Comment:
Why python2 here @ShadoWander

Reply:
I use some libs in python2 in this script
If we can only use python3 I can change to use the libs in python3

Reply:
I use some libs in python2 in this script
If we can only use python3 I can change to use the libs in python3

Reply:
You should create conda environment following this: https://github.com/usc-isi-i2/datamart#datamart
Update necessary packages in environment.yml and requirements.txt